src/Config/Utilities/FileUtilities.cs

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Azure.DataApiBuilder.Config.Utilities
+{
+    internal class FileUtilities
+    {
+        public static byte[] ComputeHash(string filePath)
+        {
+            int runCount = 1;
```

Comment:
what is runCount here?

Reply:
Used for exponential backoff mechanism for when file access results in an IO exception. This code is taken from example provided by dotnet team in the Change Token docs. Added comments to reference this. https://learn.microsoft.com/en-us/aspnet/core/fundamentals/change-tokens?view=aspnetcore-8.0#:~:text=exponential%20back%2Doff.-,Utilities/Utilities.cs%3A,-C%23